gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -227,16 +227,16 @@
               <label translate>Folder Type</label>
               &nbsp;<a href="{{docsURL('users/foldertypes')}}" target="_blank"><span class="fas fa-question-circle"></span>&nbsp;<span translate>Help</span></a>
               <select class="form-control" ng-change="setDefaultsForFolderType()" ng-model="currentFolder.type" ng-disabled="editingFolderExisting() && currentFolder.type == 'receiveencrypted'">
-                <option value="sendreceive" translate>Send &amp; Receive</option>
-                <option value="sendonly" translate>Send Only</option>
-                <option value="receiveonly" translate>Receive Only</option>
-                <option value="receiveencrypted" ng-disabled="editingFolderExisting()" translate>Receive Encrypted</option>
+                <option value="sendreceive">{{friendlyNameFromFolderType('sendreceive').replace('&amp;amp;', '&amp;')}}</option>
```

Comment:
Why don't we move that `replace()` call inside the controller function directly? It seems to not hurt if used on a different string, and much more useful than putting a comment there that something needs remembering in different places.

Reply:
Removed the controller part, moving the translations back to HTML, so this is irrelevant as of now.